src/main/resources/org/broadinstitute/hellbender/tools/copynumber/case_denoising_calling.py

```diff
@@ -75,6 +75,13 @@
                    default=argparse.SUPPRESS,
                    help="(advanced) Path to saved optimizer state to take as the starting point")
 
+group.add_argument("--num_samples_for_copy_ratio_approx",
```

Comment:
See above about the argument name. Also make the same `denoised copy number ratios` -> `denoised copy ratios` change here.

Reply:
Done